src/tools/pontoon/renaming_a_project.md

```diff
@@ -0,0 +1,12 @@
+# Renaming a project
+
+You need to think carefully on the name when setting up a new project. Though rare, sometimes a project needs to be renamed because the product name has changed or for some other reasons. To rename it, it’s quite simple:
+
+Go to the project Admin page, update the `Name` field and click the `SAVE PROJECT` button. That’s it! There is no interruption between Pontoon and the project repo. Do send a communication on the name change to the communities.
+
+Word of caution: do NOT change the `Slug` field to make it match the revised project name. The reasons are:
+* A slug is a unique ID to the project. It doesn’t have to match the project name 100%.
```

Comment:
You should at least mention what a slug is 
https://en.wikipedia.org/wiki/Clean_URL#Slug